optimum/onnxruntime/modeling_ort.py

```diff
@@ -833,3 +834,86 @@ def forward(
         return ImageClassifierOutput(
             logits=torch.from_numpy(outputs[self.model_outputs["logits"]]),
         )
+
+
+CUSTOM_TASKS_EXAMPLE = r"""
+    Example of custom tasks(e.g. a sentence transformers taking `pooler_output` as output):
+
+    ```python
+    >>> from transformers import {processor_class}
+    >>> from optimum.onnxruntime import {model_class}
+    >>> import torch
+
+    >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
+    >>> model = {model_class}.from_pretrained("{checkpoint}")
+
+    >>> inputs = tokenizer("I love burritos!", return_tensors="pt")
+
+    >>> outputs = model(**inputs)
+    >>> last_hidden_state = outputs.last_hidden_state
+    >>> pooler_output = outputs.pooler_output
+    ```
+
+    Example using `transformers.pipelines`(only if the task is supported):
+
+    ```python
+    >>> from transformers import {processor_class}, pipeline
+    >>> from optimum.onnxruntime import {model_class}
+
+    >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
+    >>> model = {model_class}.from_pretrained("{checkpoint}")
+    >>> onnx_extractor = pipeline("feature-extraction", model=model, tokenizer=tokenizer)
+
+    >>> text = "I love burritos!"
+    >>> pred = onnx_extractor(text)
+    ```
```

Comment:
Does that make sense? Since have the [ORTModelForFeautreExtraction](https://huggingface.co/docs/optimum/onnxruntime/modeling_ort#optimum.onnxruntime.ORTModelForFeatureExtraction)?

Reply:
The example here is a [sentence BERT with `last_hidden_state` and `pooler_output`](https://huggingface.co/optimum/sbert-all-MiniLM-L6-with-pooler) as outputs. This can be traced back to [an issue](https://github.com/huggingface/optimum/issues/224) where the user wanted to leverage the `pooler_output` instead of the `last_hidden_state`. The [ORTModelForFeautreExtraction](https://huggingface.co/docs/optimum/onnxruntime/modeling_ort#optimum.onnxruntime.ORTModelForFeatureExtraction) only takes `last_hidden_state` as output so the example with pooler is somehow custom(but maybe not that custom...). WDYT? Otherwise, do you have a better example in mind?

Reply:
Nice!